### PR TITLE
✨ Added navbar

### PR DESCRIPTION
**Pull request by shaksham08**
Added Navbar

### CODE REVIEW

File: src/components/NavBar/index.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import { Link } from "react-router-dom";
+import "./index.less";
+import LogoPng from "images/MFLOGOTEXT.png";
+
+const Navbar = () => {
+  return (
+    <header>
+      <nav class="nav">
+        <div>
+          <Link class="nav__brand" to="/">
```
Comment (Mohitkumar33) [inline src/components/NavBar/index.jsx]: change the class to className in whole project

File: src/index.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import ReactDOM from "react-dom";
-import "./index.css";
+import "./index.less";
```
Comment (Mohitkumar33) [inline src/index.js:3]: seen something new. Nice

**Review by Mohitkumar33 — APPROVED** [review]
Overall clean code.

File: src/components/NavBar/index.jsx (hunk 1 above)
Comment (shaksham08) [reply src/components/NavBar/index.jsx]: thanks i missed it

File: src/components/NavBar/index.less [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  min-height: 10vh;
+  box-shadow: 0px 4px 8px -8px var(--black-primary);
+  -webkit-box-shadow: 0px 4px 8px -8px var(--black-primary);
+  -moz-box-shadow: 0px 4px 8px -8px var(--black-primary);
+}
+
+.nav__link i {
+  margin-bottom: 0.5rem;
+  margin-left: 0.5rem;
+}
```
Comment (sushil-mod) [inline src/components/NavBar/index.less:18]: this is fine ,you can also use  margin : 0 0 0.5rem 0.5rem

**Review by sushil-mod — APPROVED** [review]
code is fine ,  i found new thing index.less . just added one  suggesstion

**Pull request merged**